tox.ini

```diff
@@ -45,13 +43,45 @@ deps =
     twcurrent: Twisted
     twtrunk: https://github.com/twisted/twisted/archive/trunk.zip
 
+    {trial,coverage}: mock
+
+    coverage: coverage
+
+
+passenv =
+    PATH
+    CI CONTINUOUS_INTEGRATION
```

Comment:
I don't see anything in this changeset which references these. Are they for Hypothesis in #181?